docker/ci/install-jupyter.bash

```diff
@@ -15,18 +15,26 @@
 
 set -e
 
-# NOTE: install-python.bash installs the Jupyter server.  Here we install only the magic command.
- 
-# Load SQLFlow's Jupyter magic command automatically. c.f. https://stackoverflow.com/a/32683001.
+# This file depends on install-python.bash.
+pip install --quiet \
+    jupyter==1.0.0 \
+    notebook==6.0.0 \
+    sqlflow==0.9.0 # sqlflow is the Python client of SQLFlow server.
+
+# Load SQLFlow's Jupyter magic command
+# automatically. c.f. https://stackoverflow.com/a/32683001.
 mkdir -p "$IPYTHON_STARTUP"
 mkdir -p /workspace
 { echo 'get_ipython().magic(u"%reload_ext sqlflow.magic")';
   echo 'get_ipython().magic(u"%reload_ext autoreload")';
-  echo 'get_ipython().magic(u"%autoreload 2")'; } >> "$IPYTHON_STARTUP"/00-first.py
```

Comment:
I reduce most long lines (>80 characters) in shell script files involved in this PR.  Such corrections make it easy to read the files but are not necessary to this change.